server/sonar-db-dao/src/test/java/org/sonar/db/component/ComponentDaoTest.java

```diff
@@ -696,7 +696,9 @@ public void insert() {
       .setDeveloperUuid("uuid_7")
       .setEnabled(true)
       .setCreatedAt(DateUtils.parseDate("2014-06-18"))
-      .setAuthorizationUpdatedAt(123456789L);
+      .setAuthorizationUpdatedAt(123456789L)
+      .setTags(newArrayList("platform", "analyzers"))
+      ;
```

Comment:
It would have been a good opportunity to get rid of the DB unit file of this unit test (not from every tests but at least on this one)